lib/bin/acars-decoder-test.ts

```diff
@@ -44,3 +44,6 @@ decode(
   'H1',
   '#M1BPOSN32492W117178,N33492W118335,195835,350,N32463W117143,195907,N32359W117014,M51,25872,249/TS195835,1214195C27'
 );
+
+// Label 8E 
+decode("8E", "EGSS,1618");
```

Comment:
This reminds me I need to get a test framework going here so that we can formalize the tests and hook it into the CI flow here, so we can catch issues. Not something you need to do (you're welcome to take a stab at it if you even want to in a followup PR, but not expected :))